sessions/nowcasting.qmd

```diff
@@ -420,8 +420,8 @@ The joint nowcast was the best of the three as it properly handled the uncertain
 - However, the joint nowcast is still quite simple and so may struggle to capture more complex patterns in the data.
```

Comment:
```suggestion
- However, the joint nowcast is still quite simple (in the sense that no detailed mechanism or reporting process is being modelled) and so may struggle to capture more complex patterns in the data.
```
maybe? I think people might find it not simple at all, even on a conceptual level